app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/SettingsFragment.java

```diff
@@ -45,8 +48,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         background_sync_status = (TextView) rootView.findViewById(R.id.background_sync_status);
         background_sync = (Switch) rootView.findViewById(R.id.background_sync);
 
-        notification.setChecked(Boolean.parseBoolean(preferences.getString(NOTIFICATION, "")));
-        background_sync.setChecked(Boolean.parseBoolean(preferences.getString(BACKGROUND_SYNC, "")));
+        notification.setChecked(preferences.getBoolean(String.valueOf(NOTIFICATION), false));
```

Comment:
@georgezsiga here the preference value should have been a boolean, so this should look like:
`preferences.getBoolean(NOTIFICATION, false)`